data/mmtf-filter.json

```diff
@@ -0,0 +1,272 @@
+[{
+    "categoryName": "cell",
+    "columnName": "length_a",
+    "behavior": "whitelist"
```

Comment:
Have you thought about using "whitelist" as a default for "behavior"? That could make the config less verbose and I would argue that users either supply a large list of whitelisted or a small list of blacklisted categories.

Reply:
Simplified, retained the functionality though. See #17 in molstar.